### PR TITLE
test(progress): updated demo page

### PR DESCRIPTION
**Pull request by ManahariDahal**
**Related Issue:** #2636 

## Summary
- updated progress demo page

### CODE REVIEW

**Review by bstifle — APPROVED** [review]
ship it! nice

File: src/demos/calcite-progress.html [hunk 1]
```diff
@@ -6,46 +6,62 @@
...
+      <div class="child">Intermediate</div>
+
+      <div class="child">
+        <calcite-progress type="indeterminate" label="a11y label"></calcite-progress>
+      </div>
+    </div>
+
+    <!-- Determinate -->
+    <div class="parent">
+      <div class="child">Determinate</div>
```
Comment (julio8a) [inline src/demos/calcite-progress.html:46]: this should read `Indeterminate`

**Review by julio8a — APPROVED** [review]
Overall looks good, requested a typo change.

I'm curious how we can test the Indeterminate with the screener, as it will likely see a difference when it takes a screenshot (Screener). It would be annoying to have to approve Screener for every PR that it records a different screenshot, same for loader probably. 

But needed for the manual testing.

File: src/demos/calcite-progress.html [hunk 2]
```diff
@@ -6,46 +6,62 @@
...
-      >
-      </calcite-progress>
+    <div>
+      <calcite-button href="/" icon-start="arrow-left" color="neutral"> Home </calcite-button>
+    </div>
+    <h1 style="margin: 0 auto; text-align: center">Progress</h1>
+
+    <!-- Intermediate -->
+    <div class="parent">
+      <div class="child">Intermediate</div>
```
Comment (julio8a) [inline src/demos/calcite-progress.html:37]: this should read `Indeterminate`

**Comment by ManahariDahal** [comment]
> Overall looks good, requested a typo change.
> 
> I'm curious how we can test the Indeterminate with the screener, as it will likely see a difference when it takes a screenshot (Screener). It would be annoying to have to approve Screener for every PR that it records a different screenshot, same for loader probably.
> 
> But needed for the manual testing.

manual testing? It would be easy if the screener doesn't scream at us.

**Comment by julio8a** [comment]
Yea, like Erik said on the loader, we can have screener ignore that and manually test.

**Comment by ManahariDahal** [comment]
wait, let me fix the typo 😅

**Comment by julio8a** [comment]
ah, my bad. Totally merged this without the typo changes. hehe

**Pull request merged**